EventsExpress/EventsExpress.csproj

```diff
@@ -91,4 +91,6 @@
     </ItemGroup>
   </Target>
 
+  <ProjectExtensions><VisualStudio><UserProperties clientapp_4_1jshintrc__JsonSchema="" /></VisualStudio></ProjectExtensions>
```

Comment:
Why is this piece required? Should we remove it?

Reply:
This line was deleted